src/ui/Pager/Pager.ts

```diff
@@ -349,7 +349,7 @@ export class Pager extends Component {
     lastPageNumber = Math.max(lastPageNumber, 1);
     const halfLength = Math.floor(this.options.numberOfPages / 2);
     let firstPageNumber = currentPage - halfLength;
```

Comment:
could we use multiple const variables instead of reassigning? Would be a tad more clear